recipe/meta.yaml

```
@@ -17,13 +17,15 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python >=3.10
     - pip
+    - setuptools >=64
+    - setuptools-scm >=7
```

```suggestion
    - setuptools-scm >=8
```

https://github.com/SciTools/iris-grib/blob/1eec6a6af05d8d40080e9669a0cedc693a4809fe/requirements/py310.yml#L11